checks.d/etcd.py

```diff
@@ -92,6 +96,16 @@ def check(self, instance):
         timeout = float(instance.get('timeout', self.DEFAULT_TIMEOUT))
         is_leader = False
 
+        # Gather self health status
+        self_response = self._get_self_health(url, ssl_params, timeout)
+        if self_response is not None:
+            for key in self.SELF_HEALTH:
+                if key in self_response:
+                    val = 1 if self_response[key] == 'true' else 0
```

Comment:
I think a service check would actually make more sense, something like `etcd.is_healthy` (first name I found, there's probably a better one) which would do something like `etcd.can_connect`. What do you think ?

(short documentation about service checks: http://docs.datadoghq.com/guides/agent_checks/#sending-service-checks)